examples/stable_diffusion_v2/README.md

```diff
@@ -34,6 +34,10 @@ For a quick tour, please view [demo](demo.md).
     - [Chinese Prompt Adaptation](#chinese-prompt-adaptation)
 - [Stable Diffusion 1.5](#stable-diffusion-15)
   - [Inference](#inference-1)
+    - [Text-to-Image Generation](#sd15-text-to-image-generation)
+    - [Chinese Text-to-Image Generation](#chinese-text-to-image-generation)
+    - [Chinese Text-guided Image Inpainting](#chinese-text-guided-image-inpainting)
+    - [Controlled Image-to-Image Generation with T2I-Adapter](#controlled-image-to-image-generation-with-t2i-adapter-1)
```

Comment:
Although it just works with sd1.x currently, considering T2I Adapter and controlnet are equally significant but the bind Sd versions varies, it would be simpler to put T2i adapter in level 1 (same level as as1.5）

Reply:
Will be the following structure good?
- Controlled Image-to-Image Generation (level 1)
  - ControlNet
  - T2I-Adapter

Reply:
Here is the structure

... 
- [Stable Diffusion 1.5](#stable-diffusion-15)
  - [Inference](#inference-1)
  - [Training](#training-1)
- [Stable Diffusion with ControlNet]()
  - [Inference]()
  - [Training]()
- [Stable Diffusion with T2I Adapter]()
  - [Inference]()
  - [Training]()

Reply:
Done. @congw729 please add docs for ControlNet in a separate PR.